<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="loginTab">
       <attribute name="title">
        <string>Login</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelNickname">
            <property name="text">
             <string>Nickname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nicknameEdit">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Enter your nickname here</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="arenaLabel">
            <property name="text">
             <string>Arena</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="arenaEdit">
            <property name="placeholderText">
             <string>Enter the arena</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="passwordLabel">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwordEdit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Enter password here</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="connectButton">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="robotTab">
       <attribute name="title">
        <string>Robot</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lifeInfoLabel">
          <property name="text">
           <string>Life</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QProgressBar" name="lifeBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lifeLabel">
          <property name="text">
           <string>0/0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="ammoInfoLabel">
          <property name="text">
           <string>Ammo</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QProgressBar" name="ammoBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="ammoLabel">
          <property name="text">
           <string>0/0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButtonUp">
            <property name="text">
             <string>haut </string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="pushButtonDown">
            <property name="text">
             <string>bas</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButtonLeft">
            <property name="text">
             <string>gauche</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButtonRight">
            <property name="text">
             <string>droite</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="autoMode">
            <property name="text">
             <string>Auto</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButtonShoot">
            <property name="text">
             <string>Tirer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonStopShoot">
            <property name="text">
             <string>stop Tirer </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>arenaEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onArenaTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nicknameEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onNicknameTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onPasswordTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonUp</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>onUpButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDown</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDownButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLeft</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>onLeftButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRight</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRightButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonShoot</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>onShootButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStopShoot</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onNotShootButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onAutoMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectButtonClicked()</slot>
  <slot>onNicknameTextChanged(QString)</slot>
  <slot>onArenaTextChanged(QString)</slot>
  <slot>onPasswordTextChanged(QString)</slot>
  <slot>onUpButtonClicked()</slot>
  <slot>onLeftButtonClicked()</slot>
  <slot>onRightButtonClicked()</slot>
  <slot>onDownButtonClicked()</slot>
  <slot>onShootButtonClicked()</slot>
  <slot>onNotShootButtonClicked()</slot>
  <slot>onAutoMode(bool)</slot>
 </slots>
</ui>
